components/comment/demo/editor.md

```diff
@@ -0,0 +1,136 @@
+---
+order: 3
+title:
+  zh-CN: 编辑模式
+  en-US: Editor mode
+---
+
+## zh-CN
+
+评论编辑器组件提供了相同样式的封装以支持自定义评论编辑器。
+
+## en-US
+
+Comment can be used as editor, user can customize the editor component.
+
+````jsx
+import { Comment, Avatar, Form, Button, List, Input } from 'antd';
+import moment from 'moment';
+
+const FormItem = Form.Item;
+const TextArea = Input.TextArea;
+
+function hasErrors(fieldsError) {
+  return Object.keys(fieldsError).some(field => fieldsError[field]);
+}
+
+function CommentList({ comments }) {
+  return (
+    <List
+      header={`${comments.length} ${comments.length > 1 ? 'replies' : 'reply'}`}
+      itemLayout="horizontal"
+      dataSource={comments}
+      renderItem={props => <Comment {...props} />}
+    />
+  );
+}
+
+class Editor extends React.Component {
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.props.form.validateFields((err, values) => {
+      if (!err) {
+        this.props.onSubmit(
+          values,
+          () => this.props.form.resetFields()
+        );
+      }
+    });
+  }
+
+  render() {
+    const { getFieldDecorator, getFieldsError } = this.props.form;
+
+    return (
+      <Form onSubmit={this.handleSubmit}>
+        <FormItem>
+          {getFieldDecorator('message', {
+            rules: [{ required: true, message: 'Please input your message!' }],
+          })(
+            <TextArea rows={4} />
+          )}
```

Comment:
Maybe we don't need Form and required validation here, it is a simple input that we can keep it simple in demo.

Reply:
@afc163 The validation can be removed, It was just important to show how comment can be used together with antd form

Reply:
So you think better to remove form completely and keep only `TextArea`?

Reply:
Right~